include/imu/imu.h

```diff
@@ -37,7 +39,7 @@ int imu_soft_init(LoggerConfig *loggerConfig);
 
 void imu_calibrate_zero();
 
-int imu_read(enum imu_channel channel);
+int imu_read_raw(enum imu_channel channel);
```

Comment:
It looks like you have some unrelated code mixed in with these changes - here and the GPS data.  Please submit these as part of separate feature PR's. Thanks!